### ISSUE TITLE
buildParameters problem

### ISSUE THREAD

**Issue by ming436534** [julienvincent/modelizr#14]
```
const buildParameters = (params: ?Object): string => {
	params = _.pickBy(params || {}, value => value !== null && value !== undefined && !Object.is(value, NaN))
	if (_.isEmpty(params)) return ""
	return `(${_.map(_.pickBy(params, param => param || param === false), (param, key) =>
		`${key}: ${JSON.stringify(param).replace(/"([^(")"]+)":/g, "$1:")}`)})`
}
```
May I ask what is the purpose of `_.pickBy(params, param => param || param === false` ? This part of code would omit my param if my param is like {page:0}. It makes problem as my pagination 0 based. May I ask if this is on purpose? Thank you.

**Comment by julienvincent** [comment]
Hmm that looks like a mistake. I think I meant to replace that line with the variable `params` created above.

Could you submit a PR?

**Comment by ming436534** [comment]
@julienvincent I have created a PR #15, thank you.

**Comment by julienvincent** [comment]
Great! Published as `1.0.5`
